examples/TestRunner/JuceLibraryCode/AppConfig.h

```diff
@@ -23,135 +23,166 @@
 /*
   ==============================================================================
 
-   In accordance with the terms of the JUCE 5 End-Use License Agreement, the
+   In accordance with the terms of the JUCE 6 End-Use License Agreement, the
    JUCE Code in SECTION A cannot be removed, changed or otherwise rendered
    ineffective unless you have a JUCE Indie or Pro license, or are using JUCE
    under the GPL v3 license.
 
-   End User License Agreement: www.juce.com/juce-5-licence
+   End User License Agreement: www.juce.com/juce-6-licence
 
   ==============================================================================
 */
 
 // BEGIN SECTION A
 
 #ifndef JUCE_DISPLAY_SPLASH_SCREEN
- #define JUCE_DISPLAY_SPLASH_SCREEN 0
-#endif
-
-#ifndef JUCE_REPORT_APP_USAGE
- #define JUCE_REPORT_APP_USAGE 1
+ #define JUCE_DISPLAY_SPLASH_SCREEN 1

```

Comment:
Will these changes survive the CMake transition? Generally feel like we can probably remove teh whole `JuceLibraryCOde` directory and the `Builds/` directory when we get to CMake

Reply:
I really couldn't say... I haven't had the time to familiarise myself with the way JUCE handles cmake.